### ISSUE TITLE
cmake version

### ISSUE THREAD

**Issue by ekg** [evotools/hapbin#2]
Would it be safe to force the build to work with an older version of cmake? As far as I can tell I have satisfied all the requirements of the system, but my cmake version is too old.

```text
CMake Error at CMakeLists.txt:18 (cmake_minimum_required):
  CMake 2.8.12 or higher is required.  You are running version 2.8.7
```

This minor-version difference is really unlikely to present a real problem. But, it blocks an easy build of the software.

**Comment by ekg** [comment]
I've opened a pull request: https://github.com/evotools/hapbin/pull/3

**Comment by camaclean** [comment]
2.6.11 is needed for the test framework. However, since that is optional, we could lower the minimum, and check the version again later. I believe 2.6.11 was encountering some bugs which were fixed in 2.6.12, as well. I believe the requirements for generating export targets require a higher version as well.

**Comment by npch** [comment]
The latest version of hapbin has downgraded the CMake version requirements to 2.6 or higher.
